docs/examples.md

```diff
@@ -196,15 +174,40 @@ job('example'){
 import jenkins.automation.utils.CommonUtils
 
 job("example"){
-  CommonUtils.addExtendedEmail(delegate, 
-  emails = 'someperson@email.com, someotherperson@email.com') 
-         
+    CommonUtils.addExtendedEmail(delegate, emails = 'foo@example.com, bar@example.com') 
 }
    
-   
 // override accepts emails as a list. Compatible with builders
 job('example'){
-  CommonUtils.addExtendedEmail(delegate, 
-  emails = ['someperson@email.com', 'someotherperson@email.com']) 
+    CommonUtils.addExtendedEmail(delegate, emails = ['foo@example.com', 'bar@example.com']) 
```

Comment:
Similar to the question about `repos`, we have the `emails` abstraction that can be added in to the constructor. Should we document that pattern instead?